download/linux.md

```diff
@@ -2,8 +2,21 @@
 title: Install MonoDevelop on Linux
 ---
 
-Install MonoDevelop on Linux
-============================
+Install MonoDevelop preview via FlatPak
+=======================================
+
+[FlatPak](http://flatpak.org/) is a new packaging format for Linux, which allows the same applications to run in a sandboxed environment, across any modern distribution.
+
+We are offering a preview release of MonoDevelop as a FlatPak bundle, which does not require Mono to be installed.
+
+[Download the preview here]({{ site.data.latestrelease.monodevelop_linux_url }}).
+
+Please follow step 1 and 2 from [here](http://flatpak.org/#users) as appropriate, and follow step 3 using the runtime `org.freedesktop.Platform`
+
+You can now install our FlatPak bundle, using `flatpak install --user --bundle monodevelop-download.flatpak`
```

Comment:
I think we could make this show the real filename by doing `{{ site.data.latestrelease.monodevelop_linux_url | split: '/' | last }}` instead of `monodevelop-download.flatpak`